cypress/integration/models/jobfunctions.ts

```diff
@@ -24,48 +27,48 @@ export class Jobfunctions {
         inputText(jobfunctionNameInput, name);
     }
 
-    getJobFuncName(): string {
-        this.jobfunctionName = faker.name.jobType();
-        return this.jobfunctionName;
-    }
-
-    create(): void {
+    create(cancel = false): void {
         Jobfunctions.clickJobfunctions();
-        clickByText("button", "Create new");
-        this.getJobFuncName();
-        this.fillName(this.jobfunctionName);
+        clickByText(button, createNewButton);
+        this.fillName(this.name);
         submitForm();
     }
 
-    edit(): void {
+    edit(updatedName: string, cancel = false): void {
         Jobfunctions.clickJobfunctions();
         selectItemsPerPage(100);
         cy.wait(2000);
         cy.get(tdTag)
-            .contains(this.jobfunctionName)
+            .contains(this.name)
             .parent(trTag)
             .within(() => {
-                click(editButton);
+                click(commonView.editButton);
             });
-        this.getJobFuncName();
-        this.fillName(this.jobfunctionName);
-        submitForm();
+        if (cancel) {
+            cancelForm();
+        } else {
+            if (updatedName != this.name) {
+                this.fillName(updatedName);
+                this.name = updatedName;
+                submitForm();
+            }
+        }
     }
 
-    delete({ jobfunctionName = this.jobfunctionName, cancel = false } = {}): void {
+    delete(cancel = false): void {
         Jobfunctions.clickJobfunctions();
```

Comment:
did we miss `name` here as optional argument. How can I delete multiple items?

Reply:
Not updated this, as Shveta will be taking care in her PR.